use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java

```diff
@@ -0,0 +1,225 @@
+                .iterator();
+        PrimitiveIterator.OfInt vehicleCapacity = random.ints(demoData.minVehicleCapacity, demoData.maxVehicleCapacity + 1)
+                .iterator();
+
+        PrimitiveIterator.OfInt depotRandom = random.ints(0, demoData.depotCount).iterator();
+
+        AtomicLong depotSequence = new AtomicLong();
+        Supplier<Depot> depotSupplier = () -> new Depot(
+                String.valueOf(depotSequence.incrementAndGet()),
+                new Location(latitudes.nextDouble(), longitudes.nextDouble()));
+
+        List<Depot> depots = Stream.generate(depotSupplier)
+                .limit(demoData.depotCount)
+                .collect(Collectors.toList());
+
+        AtomicLong vehicleSequence = new AtomicLong();
+        Supplier<Vehicle> vehicleSupplier = () -> new Vehicle(
+                String.valueOf(vehicleSequence.incrementAndGet()),
+                vehicleCapacity.nextInt(),
+                depots.get(depotRandom.nextInt()),
+                tomorrowAt(demoData.vehicleStartTime));
+
+        List<Vehicle> vehicles = Stream.generate(vehicleSupplier)
+                .limit(demoData.vehicleCount)
+                .collect(Collectors.toList());
+
+        Supplier<String> nameSupplier = () -> {
+            Function<String[], String> randomStringSelector = strings -> strings[random.nextInt(strings.length)];
+            String firstName = randomStringSelector.apply(FIRST_NAMES);
+            String lastName = randomStringSelector.apply(LAST_NAMES);
+            return firstName + " " + lastName;
+        };
+
+        AtomicLong customerSequence = new AtomicLong();
+        Supplier<Customer> customerSupplier = () -> {
+            boolean morningTimeWindow = random.nextBoolean();
+
+            LocalDateTime minStartTime = morningTimeWindow ? tomorrowAt(MORNING_WINDOW_START) : tomorrowAt(AFTERNOON_WINDOW_START);
+            LocalDateTime maxEndTime = morningTimeWindow ? tomorrowAt(MORNING_WINDOW_END) : tomorrowAt(AFTERNOON_WINDOW_END);
+            int serviceDurationMinutes = SERVICE_DURATION_MINUTES[random.nextInt(SERVICE_DURATION_MINUTES.length)];
+            return new Customer(
+                    String.valueOf(customerSequence.incrementAndGet()),
+                    nameSupplier.get(),
+                    new Location(latitudes.nextDouble(), longitudes.nextDouble()),
+                    demand.nextInt(),
+                    minStartTime,
+                    maxEndTime,
+                    Duration.ofMinutes(serviceDurationMinutes));
+        };
+
+        List<Customer> customers = Stream.generate(customerSupplier)
+                .limit(demoData.customerCount)
+                .collect(Collectors.toList());
+
+        return new VehicleRoutePlan(name, demoData.southWestCorner, demoData.northEastCorner, tomorrowAt(demoData.vehicleStartTime), tomorrowAt(LocalTime.MIDNIGHT).plusDays(1L), depots, vehicles, customers
+        );
```

Comment:
The length of this line suggests to me that your IDE is not configured properly.
(Quickstarts don't have automated formatting via Maven plugin.)

Reply:
That could be the case :) I will ping you about IDE config recommendation.

Reply:
Done.